src/main/java/hudson/plugins/ec2/win/winrm/request/OpenShellRequest.java

```diff
@@ -13,7 +13,7 @@
 
 import com.google.common.collect.ImmutableList;
 
-public class OpenShellRequest extends AbstractWinRMRequest implements WinRMRequest {
+public class OpenShellRequest extends AbstractWinRMRequest {
```

Comment:
AbstractWinRMRequest already implements WinRMRequest as such this can be removed